h2/connection.py

```diff
@@ -1429,6 +1473,9 @@ def receive_data(self, data):
         :returns: A list of events that the remote peer triggered by sending
             this data.
         """
+        self.config.logger.info("Process received data on connection")
```

Comment:
This should be DEBUG, not INFO. You can therefore probably merge these two logs.